app/services/integrations/aggregator/taxes/credit_notes/payload.rb

```diff
@@ -51,7 +51,7 @@ def cn_item(item)
             {
               'item_id' => fee.item_id,
               'item_code' => mapped_item.external_id,
-              'amount_cents' => (fee.sub_total_excluding_taxes_amount_cents&.to_i || 0) * -1
+              'amount_cents' => (item.amount_cents&.to_i || 0) * -1
```

Comment:
I am pretty sure we should send value without coupons. Can you double check please?

Reply:
we can issue credit note for the amount less than the original fee.
<img width="779" alt="image" src="https://github.com/user-attachments/assets/c5179132-d533-44fa-834f-cadd497c4e15">
in this case the value that is credited for this exact fee is stored in the credit_note_item.amount_cents.
so we should take the anrok details from the fee, but the exact amount that is being send as credit_note is stored in the item.
As well as sum of the credit note amount can be less then the invoice sum, even if we partially return all fees from the invoice

Reply:
Previous (Existing) line seems invalid, probably that is some leftover from invoice logic. Good catch!

However, It seems to me that new logic is also not valid. Item amount cents include value without coupons included in that calculation. It seems to me that we should also take coupons into account. You can try to reproduce the case with applying coupons on customer -> issue invoice -> issue CN. I am not sure if this case is covered

Reply:
I see. Didn't know that we apply coupons proportionally in the credit notes.
Will add the calculation logic then

Reply:
added a method similar to the one we have in `credit_note` and `fee` that proportionally calculates item.sub_total_excluding_taxes_amount_cents, thank you!